boards/prototype/base-workloads/br-chip-base/linux-config

```diff
@@ -0,0 +1,60 @@
+#CONFIG_BLK_DEV_INITRD=y
+#CONFIG_EMBEDDED=y
+#CONFIG_HZ_100=y
```

Comment:
Have you confirmed that these are not set in the final config? My understanding is that to disable something that was set by defconfig you have to do: 
`# CONFIG_HZ_100 is not set`

or
`CONFIG_HZ_100=n`

I wonder if I should change marshal to use defnoconfig instead of defconfig and have the base workloads specify way more stuff...

Reply:
I don't think they are getting set (though I haven't explicitly checked). I still need to clean this up though. 

As for moving to `defnoconfig`... that is definitely an option but wouldn't that make it harder to port to newer Linux versions?

Reply:
Ya, the defnoconfig would definitely be annoying which is why I don't do it. So far it looks like we're fine with it as is.